zcash_graphs/analysis/analyze.py

```diff
@@ -113,7 +128,7 @@ def aggregate(self, kvs):
         kvs.sort(key=lambda x: x[0])
         return self.__group(kvs, [x[1] for x in self.__bucketers])
 
-    def __group(self, kvs, accumulators):
+    def __group(self, kvs: list[tuple[list[Key], Value]], accumulators):
```

Comment:
```suggestion
    def __group(
            self,
            kvs: list[tuple[list[Key[_]], Value]],
            accumulators: list[Callable[[XDict[Key[1], Result[1]] | list[Value]], Result[0]]],
    ) -> XDict[Key[0], Result[0]]:
```
`kvs` and the result are the same as in `aggregate`, and the `accumulators` are the second elements of the `bucketers` tuples (with the `outer_` and `innermost_` combined here).